apps/builder/app/builder/features/settings-panel/props-section/animation/animation-panel-content.tsx

```diff
@@ -114,7 +119,14 @@ const RangeValueInput = ({
       intermediateValue={intermediateValue}
       onChange={(styleValue) => {
         setIntermediateValue(styleValue);
-        /* @todo: allow to change some ephemeral property to see the result in action */
+
+        const parsedValue = rangeUnitValueSchema.safeParse(styleValue);
```

Comment:
Is it instead of styleValue.type check?

Reply:
We are using a little bit more strict types for ranges etc.